deploy/example_clusters/cr-cluster-cdh632cm-stor.yaml

```diff
@@ -0,0 +1,104 @@
+apiVersion: "kubedirector.hpe.com/v1beta1"
+kind: "KubeDirectorCluster"
+metadata:
+  name: "cdh632-cm-persistent"
+spec:
+  app: cdh632
+  roles:
+  - id: cmserver
+    resources:
+      requests:
+        memory: "8Gi"
+        cpu: "4"
+      limits:
+        memory: "8Gi"
+        cpu: "4"
+    storage:
+      size: "100Gi"
+    podLabels:
+      hpecp.hpe.com/dtap: "inject"
+  - id: primary
+    resources:
+      requests:
+        memory: "4Gi"
+        cpu: "4"
+      limits:
+        memory: "4Gi"
+        cpu: "4"
+    storage:
+      size: "100Gi"
+    podLabels:
+      hpecp.hpe.com/dtap: "inject"
+  - id: hive
+    resources:
+      requests:
+        memory: "2Gi"
+        cpu: "2"
+      limits:
+        memory: "2Gi"
+        cpu: "2"
+    storage:
+      size: "100Gi"
+    podLabels:
+      hpecp.hpe.com/dtap: "inject"
+  - id: worker
+    members: 3
+    resources:
+      requests:
+        memory: "4Gi"
+        cpu: "2"
+      limits:
+        memory: "4Gi"
+        cpu: "2"
+    storage:
+      size: "100Gi"
+    podLabels:
+      hpecp.hpe.com/dtap: "inject"
+  - id: gateway
+    resources:
+      requests:
+        memory: "4Gi"
+        cpu: "2"
+      limits:
+        memory: "4Gi"
+        cpu: "2"
+    storage:
+      size: "100Gi"
+    podLabels:
+      hpecp.hpe.com/dtap: "inject"
+  - id: hbase
+    resources:
+      requests:
+        memory: "2Gi"
+        cpu: "2"
+      limits:
+        memory: "2Gi"
+        cpu: "2"
+    storage:
+      size: "100Gi"
+    podLabels:
```

Comment:
@omhpe 
I missed this as well in our discussions. I think we need to remove dtap podlabels from the cluster CR as dtap is an EPIC concept. If users want dtap functionality, they can add the podlabels manually.

Right @swamibluedata and @joel-bluedata ?

Reply:
Right.

Reply:
@riteshja @neogeographica  Sure, will make the change.

Reply:
That was me above FYI. Wrong account logged in. :-)